tests/tests.js

```diff
@@ -1,14 +1,119 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const {
+  expect
+} = require('chai')
+const {
+  describe,
+  it
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
+describe('formatPrice', () => {
+  it('returns price as an integer formatted to two decimal places', () => {
+    const price = pricing.formatPrice(21.7897)
+
+    expect(price).to.equal(21.78)
+  })
+})
+
+describe('getEmployerContribution', () => {
+  it('returns reduced product price based upon employer contribution for percentage mode', () => {
+    const dollarsOff = pricing.getEmployerContribution(products.voluntaryLife.employerContribution, 39.37)
+
+    expect(dollarsOff).to.equal(3.937)
+  })
+
+  it('returns employer contribution based upon employer contribution for dollars mode', () => {
+    const dollarsOff = pricing.getEmployerContribution(products.commuter.employerContribution, 39.37)
+
+    expect(dollarsOff).to.equal(75)
+  })
+})
+
+describe('calculateVolLifePricePerRole', () => {
+  it('returns volLife price for a single employee', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['ee'],
+      coverageLevel: [{
+        role: 'ee',
+        coverage: 125000
+      }],
+    }
+    const price = pricing.calculateVolLifePricePerRole(selectedOptions.coverageLevel[0].role, selectedOptions.coverageLevel, products.voluntaryLife.costs)
+
+    expect(price).to.equal(43.75)
+  })
+
+  it('returns volLife plan price for a employee with a spouse', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['sp'],
+      coverageLevel: [{
+        role: 'sp',
+        coverage: 85000
+      }],
+    }
+
+    const price = pricing.calculateVolLifePricePerRole(selectedOptions.coverageLevel[0].role, selectedOptions.coverageLevel, products.voluntaryLife.costs)
+
+    expect(price).to.equal(10.2)
+  })
+})
+
+describe('calculateVolLifePrice', () => {
+  it('returns price for volLife plan for single employee', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['ee'],
+      coverageLevel: [{
+        role: 'ee',
+        coverage: 125000
+      }],
+    }
+
+    const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+    expect(price).to.equal(43.75)
+  })
+
+  it('returns price for volLife plan for an employee with a spouse', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['sp'],
+      coverageLevel: [{
+        role: 'sp',
+        coverage: 125000
+      }],
+    }
+
+    const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+    expect(price).to.equal(15)
+  })
+})
```

Comment:
This test is testing volLife for just a spouse since there is no `ee` in family members to cover. See original test below.